mona_openai/mona_openai.py

```diff
@@ -72,6 +73,21 @@ def _get_mona_single_message(
     )
 
 
+def _init_mona_class(client, context_class_name, openai_endpoint_name):
+    response = client.create_openai_context_class(
+        context_class_name, openai_endpoint_name
+    )
+    error_message = response.get("error_message")
+    if error_message:
+        logging.warning(
+            f"Problem initializing Mona context class '{context_class_name}':"
+            f" {error_message}"
+        )
+    else:
+        logging.info("Made sure Mona context class is initialised")
+    return response
```

Comment:
This is intentional. Even if init didn't work, I still won't to try and send the data, so that we could internally fix the issue and not lose the logged out information.